tests/components/digital_ocean/test_init.py

```diff
@@ -0,0 +1,53 @@
+from homeassistant.components.digital_ocean import DOMAIN
```

Comment:
I get your point, but I don't think it will be THAT beneficial in this case, as it's merely 3 test cases.

I was not even going to add init tests, but then I realized that I did modify the init to register the new service, so I ended up adding them.